### PR TITLE
Update templates

### PR DESCRIPTION
**Pull request by Sunjammer**
This PR implements the following template changes:

* New app template with folder structure and model js
* New camera app template
* New master/detail template
* New Empty template (the current app template)
* New "package" template (empty with unoproj with IsTransitive)

The PR depends on https://github.com/fusetools/Fuse/pull/4023

### CODE REVIEW

File: ProjectTemplates/Projects/App-Camera/Pages/Lightbox/LightBoxPage.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+var App = require("_$filename$_");
+var images = App.getPictures();
+module.exports = {
+	images : images,
+	currentIndex : this.Parameter.map(function(param){
+		var idx = images.length;
```
Comment (nekronos) [inline ProjectTemplates/Projects/App-Camera/Pages/Lightbox/LightBoxPage.js]: Looks like `idx` is never used?

Comment (Sunjammer) [inline ProjectTemplates/Projects/App-Camera/Pages/Lightbox/LightBoxPage.js]: Yeah it was used for a reverse while over the images at one point, removed now :)

**Comment by kristianhasselknippe** [comment]
@Sunjammer: I think we should use ClientPanel instead of DockPanel + TopBar-/BottomBarBackground

**Comment by Sunjammer** [comment]
Can you set the background on the statusbar with clientpanel?

**Comment by kristianhasselknippe** [comment]
No i don't think so

**Comment by Sunjammer** [comment]
Then we have a problem. You pretty much always have to set the background color on android or you'll get invisible statusbar icons.

**Comment by kristianhasselknippe** [comment]
Then i agree we should keep the templates as is :)

**Comment by Sunjammer** [comment]
I should probably add in the background color though to make the reasoning clear

**Comment by COCPORN** [comment]
@Sunjammer I added a pull request to your repo from my "ign97"-user which cleans up JS and UX formatting. Will be back with some general comments on the templates.

**Comment by Sunjammer** [comment]
Ruh roh, looks like we got ourselves an _illegal_

**Comment by danhollick** [comment]
Is there anything happening with these templates? We need to know whether or not to accommodate space for these in the dashboard.

**Comment by kusma** [comment]
@danhollick: Dunno. This PR stalled due to lacking reviews and CLA-signing. The latter has been fixed, and if someone who feels they know what this is about would review it, I'd be happy to merge it.

**Comment by Sunjammer** [comment]
These templates need to be redone to reflect new best practises. They're sadly outdated already.

**Comment by kusma** [comment]
Then I suggest we close this PR, and reopen once that's fixed...

**Pull request closed without merging**